.github/workflows/ci.yml

```
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ 2.3, 3.2 ] # oldest and latest CRuby
+        ruby: [ 2.3, 3.3 ] # oldest and latest CRuby
```

```suggestion
        ruby: [ 2.3, ruby ] # oldest and latest CRuby
```
This way we don't need to manually update it anymore